vnext/Microsoft.ReactNative.Managed.CodeGen.UnitTests/CodeGen/Lkg/Microsoft.ReactNative.Managed.CodeGen.UnitTests.CodeGen.CodeGenMethodTests--IntParamAndDoublePromiseParamReturnVoid--Sync.lkg

```diff
@@ -2,6 +2,4 @@ moduleBuilder.AddSyncMethod("MyMethod", (global::Microsoft.ReactNative.IJSValueR
 {
     global::Microsoft.ReactNative.Managed.JSValueReader.ReadArgs(reader, out int arg0, out global::Microsoft.ReactNative.Managed.IReactPromise<string> arg1);
     module.Method(arg0, arg1, new global::Microsoft.ReactNative.Managed.ReactPromise<int>(writer, resolve, reject));
-}
-
-);
\ No newline at end of file
+});
```

Comment:
i think all these files are generated by the codegen, we should fix the codegen to output the right formatting?

Reply:
Right, I don't know why the old code-gen produced such odd output (with the extra newlines before the ending semicolon) but as far as I can tell all I did was update from .NET 3.1 to 6.0 and the output changed.